test/unit/modules/modules.spec.ts

```diff
@@ -253,3 +253,22 @@ test("export as specifier shouldn't effect local vars", () => {
         a = 6;
     `.expectToMatchJsResult();
 });
+
+test("export modified in for in loop", () => {
+    util.testModule`
+        export let foo = '';
+        for (foo in { x: true }) {}
+    `
+        .setReturnExport("x")
```

Comment:
Isn't it supposed to check `foo`?

Reply:
Good catch

Reply:
Fixed 770d5ffdc5cd1021fab3a248f0e391fd88307a27